README.md

```diff
@@ -118,6 +118,7 @@ Additionally, as part of this change the existing v3.x extension has been moved
 
 | Version |  Application Insights | React     | Branch
 |---------|-----------------------|-----------|-----------
+| 19.3.5  | ^3.3.6                | >= 19.0.0 | [main](https://github.com/microsoft/applicationinsights-react-js)
```

Comment:
This release won't ever direct exist, so we should leave this out until we actually have the release PR

Reply:
i.e. This PR won't be used to publish any releases, we will have a follow up PR to bump the version to 19.3.6 -- this one is using 19.3.5 so that the nightly process will create a 19.3.6-nightly release